CONTRIBUTING.md

```diff
@@ -1,11 +1,57 @@
-FIXME iirc, @tswicegood had some good pointers on good defaults for CONTRIBUTING.md
+:tipping_hand_woman: AtJSON has a [Code of Conduct](https://github.com/CondeNast-Copilot/atjson/blob/latest/CODE_OF_CONDUCT.md) that we expect all of our contributors to abide by, please check it out before contributing!
 
-This is a Lerna repository, with a bunch of sub-modules. To build and get started, do the following:
+***
 
+AtJSON is comprised of a bunch of packages, monorepo style. We use [:dragon:lerna](https://lernajs.io) to manage these dependencies.
+
+:computer: To get started, clone atjson onto your computer and navigate into the project.
+
+```bash
+git clone https://github.com/CondeNast-Copilot/atjson.git
+cd atjson
+```
+
+Now install the dependencies :sparkles:
+
+```bash
+npm install
```

Comment:
`npm -g learna` ?

Reply:
I'm not convinced that assuming a global install of `lerna` and `typescript` is preferable to using what's specified in the root `package.json`. We ensure the version in use and save ourselves some potential future headaches by using the local install. Thoughts?

Reply:
true, so maybe a wrapper script in the `package.json`?
```
script: {
  'lerna': 'lerna bootstrap  --hoist'
}
``` 
or something like that

Reply:
https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b

Reply:
Yeah, this sounds alright. Let's do a follow-up to this PR with changes here. I think this is good as a first pass. Does that sound alright?

Reply:
I'd err on the side of local lerna, but we should have `npm scripts` hooks so that we're not telling people to type things like `./node_modules/.bin/lerna`